regression/Makefile

```diff
@@ -82,18 +82,27 @@ $(INITRAMFS)/benchmark:
 $(INITRAMFS_EMPTY_DIRS):
 	@mkdir -p $@
 
+.PHONY: $(INITRAMFS)/opt/syscall_test
 $(INITRAMFS)/opt/syscall_test:
 	@make --no-print-directory -C syscall_test
 
+.PHONY: $(INITRAMFS_IMAGE)
+$(INITRAMFS_IMAGE): initramfs_filelist
+	@if ! cmp -s $(INITRAMFS_FILELIST) $(INITRAMFS_FILELIST).old ; then \
+		echo "Generating the initramfs image..."; \
+		cp $(INITRAMFS_FILELIST) $(INITRAMFS_FILELIST).old; \
+		(cd $(INITRAMFS); find . | cpio -o -H newc | gzip) > $@; \
+	fi
+
+.PHONY: initramfs_filelist
```

Comment:
I think the name `initramfs_filelist` is redundant, You can just use `$(INITRAMFS_FILELIST)`